packages/docs-reanimated/docs/core/makeMutable.mdx

```diff
@@ -0,0 +1,153 @@
+---
+sidebar_position: 8
+---
+
+# makeMutable
+
+:::caution
+
+The usage of makeMutable is discouraged in most cases. It's recommended to use the [useSharedValue](/docs/core/useSharedValue) hook instead unless you know what you're doing and are aware of the consequences (see the [Remarks](#remarks) section).
+
+makeMutable is used internally and its behavior may change over time.
+
+:::
+
+`makeMutable` is a function internally used by the [useSharedValue](/docs/core/useSharedValue) hook to create a [shared value](/docs/fundamentals/glossary#shared-value).
+
+It makes it possible to create shared values without the use of the hook, which can be useful in some cases (e.g. in the global scope, as an array of shared values, etc.).
```

Comment:
Technically, objects created with `makeMutable` are mutable values, not shared values.